features/location/impl/src/main/kotlin/io/element/android/features/location/impl/location/LocationUpdatesFlowImpl.kt

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package io.element.android.features.location.impl
+package io.element.android.features.location.impl.location
```

Comment:
nit: not a fan of `location.impl.location` as a name

Could this just go in `location.impl`, and move the send location stuff into `location.send` or `location.sending` or something?

Or a `....location.impl.common` package that could hold this + mapview?

Reply:
yeah it's un unlikely coincidence but the first `location` refers to the feature, the second one refers to the class.
Yeah perhaps the location object could go to `common`.